*** include/osgwTools/TransparencyUtils.h	2011-03-21 15:20:00.000000000 -0700
--- include/osgwTools/TransparencyUtils.h	2013-04-07 17:50:45.023676772 -0700
*************** is disabled later, if the name matches,
*** 63,68 ****
--- 63,77 ----
  static std::string s_magicStateSetName( "TransparentDeleteMe" );
  
  
+ /** \brief Returns True if the given StateSet is configured like one of our transparent StateSets.
+ \It returns True if the Node has a StateSet, and the StateSet has the following signature:
+ \li A BlendColor StateAttribute
+ \li A BlendFunc StateAttribute
+ \li GL_BLEND is enabled
+ \li Rendering hint set to TRANSPARENT_BIN
+ */
+ OSGWTOOLS_EXPORT bool isTransparent( const osg::StateSet* stateSet );
+ 
  
  /** \brief Sets a transparent StateSet on the given Node or Drawable, saving 
  its current StateSet as UserData. If nodeOrDrawable's UserData is NULL, this 
*************** bool transparentDisable( T* nodeOrDrawab
*** 170,185 ****
      return( true );
  }
  
- /** \brief Returns True if the given StateSet is configured like one of our transparent StateSets.
- \It returns True if the Node has a StateSet, and the StateSet has the following signature:
- \li A BlendColor StateAttribute
- \li A BlendFunc StateAttribute
- \li GL_BLEND is enabled
- \li Rendering hint set to TRANSPARENT_BIN
- */
- OSGWTOOLS_EXPORT bool isTransparent( const osg::StateSet* stateSet );
- 
- 
  
  /** \brief Finds StateSets with nominal transparency and marks the transparent state as PROTECTED
  
--- 179,184 ----
